test/rules/no-restricted-globals.js

```diff
@@ -0,0 +1,19 @@
+module.exports = {
+	valid: [
+		"var regex1 = /\\w+/;"
```

Comment:
Jessie also omits the regexp literal syntax, i.e., `/.../`, so this case should be an error too.

Reply:
Ah good point, I added this as another rule